pytest_reportportal/plugin.py

```diff
@@ -8,26 +8,28 @@
 
 
 def pytest_sessionstart(session):
-    PyTestService.init_service(
-        project=session.config.getini('rp_project'),
-        endpoint=session.config.getini('rp_endpoint'),
-        uuid=session.config.getini('rp_uuid'),
-        log_batch_size=int(session.config.getini('rp_log_batch_size')),
-        ignore_errors=bool(session.config.getini('rp_ignore_errors')),
-        ignored_tags=session.config.getini('rp_ignore_tags'),
-    )
-
-    PyTestService.start_launch(
-        session.config.option.rp_launch,
-        tags=session.config.getini('rp_launch_tags'),
-        description=session.config.getini('rp_launch_description'),
-    )
-
-
-def pytest_sessionfinish():
-    # FixMe: currently method of RP api takes the string parameter
-    # so it is hardcoded
-    PyTestService.finish_launch(status='RP_Launch')
+    if session.config.getoption('--collect-only', default=False) is False:
```

Comment:
Here is my thoughts. 
I am OK with this change, but personally I prefer do not increase indentation level. 
Here is example how to do this in such way: 
```python
def pytest_sessionstart(session):
    if session.config.getoption('--collect-only', default=False):
         return
    PyTestService.init_service(
        project=session.config.getini('rp_project'),
        endpoint=session.config.getini('rp_endpoint'),
        uuid=session.config.getini('rp_uuid'),
        log_batch_size=int(session.config.getini('rp_log_batch_size')),
        ignore_errors=bool(session.config.getini('rp_ignore_errors')),
        ignored_tags=session.config.getini('rp_ignore_tags'))
```